src/EventLogExpert.Eventing/EventResolvers/EventProviderDatabaseEventResolver.cs

```diff
@@ -166,19 +161,17 @@ public DisplayEventModel Resolve(EventRecord eventRecord, string OwningLogName)
 
             try
             {
-                foreach (var dbContext in dbContexts)
+                foreach (var dbContext in _dbContexts)
                 {
-                    var providerDetails = dbContext.ProviderDetails.FirstOrDefault(p => p.ProviderName.ToLower() == eventRecord.ProviderName.ToLower());
+                    var providerDetails = dbContext.ProviderDetails?.FirstOrDefault(p => p.ProviderName.Equals(eventRecord.ProviderName, StringComparison.CurrentCultureIgnoreCase));
```

Comment:
Should this be OrdinalIgnoreCase? Not sure we want this comparison changing based on current culture.